src/Strings.php

```diff
@@ -158,9 +158,12 @@ public static function normalize($input)
      */
     public static function onlyNumbers($string)
     {
+        if (is_null($string)) {
+            return "";
+        }
         return preg_replace("/[^0-9]/", "", $string);
     }
```

Comment:
Dessa forma trata quando é `null`, mas imagino que tb possa acontecer o mesmo quando `$string` for `int` ou `float`, então poderia tratar todas as 3 formas com 1 solução só, sugeriria usar:

`return preg_replace("/[^0-9]/", "", (string)$string);`

Assim funcionará para os 3 casos.

Reply:
Segue como ficaria o código de acordo com a sugestão do @gersonfs, caso faça sentido é só aprovar.

```suggestion
        return preg_replace("/[^0-9]/", "", (string)$string);
    }
```